### DISCUSSION TITLE
Resource page blank in production with no errors

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [filamentphp/filament#6784] [General]
Hi All,

Any ideas how to troubleshoot resource page being displayed as a complete blank? The whole project works as expected locally, it also worked as expected in production. No in production one of the resources is showing as a complete blank page without any errors.

Any ideas?

Thanks!

**Comment by aurawindsurfing** [comment]
UPDATE: this problem is present only in Safari. Firefox and Chrome on MacOS open it without issues.

**Comment by aurawindsurfing** [comment]
UPDATE: Removing website data in Privacy Settings solved the issue.

Any ideas @danharrin ?

**Reply by danharrin** [reply]
No ideas, no. Sounds like a very unique case

**Reply by aurawindsurfing** [reply]
Thank you I will report if I'm able to replicate it
